cinema/serializers.py

```diff
@@ -1,24 +1,63 @@
 from rest_framework import serializers
+from cinema.models import Actor, Genre, CinemaHall, Movie
 
-from cinema.models import Movie
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name"]
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row"]
```

Comment:
Yeah, it's a way easier to do it in this way, however we'll learn it in API Serializers topic, thus you have to implement it via a usual Serializer ;)